src/utils/cachedFetch.js

```diff
@@ -15,7 +15,7 @@ export const cachedFetch = (url, transformFn, cache = true) => {
   const promise = new Promise((resolve, reject) => {
     xhr.responseType = 'json'
     xhr.open('GET', url)
-    if (isProd()) {
+    if (isProd() && !isRequestHeaderDisabled()) {
```

Comment:
Does it make sense to have both conditions here?

Reply:
agree with having only one condition here.